P2-issue-disposal.md

```diff
@@ -0,0 +1,57 @@
+Github Issue Disposal Policy
+-------------------------------------------------------
+* Author(s): makdharma
+* Approver: a11r
+* Status: Draft
+* Implemented in: N/A
+* Last updated: 2017-06-15
+* Discussion at: N/A
+
+## Abstract
+
+There are hundreds of open issues in the 3 gRPC repos, some open for over 2 years. gRPC team periodically spends valuable time scrubbing the issue list. This proposal lays down concrete criteria for disposing of issues.
+
+## Background
+
+The github Issues database serves many purposes.
+
+* Capture bugs
+* Keep track of TODO
+* Submit feature requests
+* Ask questions (ad hoc documentation)
```

Comment:
It is historically used for questions, but continuation of such use should be discourage IMO, especially in light of the new bug template.